source/install/install-mmte-helm-gitlab-helm.rst

```diff
@@ -0,0 +1,238 @@
+    - name: MM_FILESETTINGS_AMAZONS3BUCKET
+      value: <MATTERMOST.MINIO.BUCKET.NAME>
+
+
+  ## Additional init containers
+  extraInitContainers: |
+    - name: bootstrap-database
+      image: "postgres:9.6-alpine"
+      imagePullPolicy: IfNotPresent
+      env:
+        - name: POSTGRES_PASSWORD_GITLAB
+          valueFrom:
+            secretKeyRef:
+              name: gitlab-postgresql-password
+              key: postgres-password
+        - name: POSTGRES_USER_GITLAB
+          value: <GITLAB.POSTGRES.USERNAME>
+        - name: POSTGRES_HOST_GITLAB
+          value:<GITLAB.POSTGRES.HOST>
+        - name: POSTGRES_PORT_GITLAB
+          value: "<GITLAB.POSTGRES.PORT>"
+        - name: POSTGRES_DB_NAME_MATTERMOST
+          value: <MATTERMOST.DATABASE.NAME>
+      command:
+        - sh
+        - "-c"
+        - |
+          if PGPASSWORD=$POSTGRES_PASSWORD_GITLAB psql -h $POSTGRES_HOST_GITLAB -p $POSTGRES_PORT_GITLAB -U $POSTGRES_USER_GITLAB -lqt | cut -d \| -f 1 | grep -qw $POSTGRES_DB_NAME_MATTERMOST; then
+          echo "database already exist, exiting initContainer"
+          exit 0
+          else
+          echo "Database does not exist. creating...."
+          PGPASSWORD=$POSTGRES_PASSWORD_GITLAB createdb -h $POSTGRES_HOST_GITLAB -p $POSTGRES_PORT_GITLAB -U $POSTGRES_USER_GITLAB $POSTGRES_DB_NAME_MATTERMOST
+          echo "Done"
+          fi
+    - name: create-minio-bucket
+      image: "minio/mc:RELEASE.2018-07-13T00-53-22Z"
+      env:
+        - name: MINIO_ENDPOINT
+          value: <GITLAB.MINIO.HOST>
+        - name: MINIO_PORT
+          value: "<GITLAB.MINIO.PORT>"
+        - name: MINIO_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: <GITLAB.MINIO.SECRET>
+              key: accesskey
+        - name: MINIO_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: <GITLAB.MINIO.SECRET>
+              key: secretkey
+        - name: MATTERMOST_BUCKET_NAME
+          value: <MATTERMOST.MINIO.BUCKET.NAME>
+      command:
+        - sh
+        - "-c"
+        - |
+          echo "Connecting to Minio server: http://$MINIO_ENDPOINT:$MINIO_PORT"
+          mc config host add myminio http://$MINIO_ENDPOINT:$MINIO_PORT $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+          /usr/bin/mc ls myminio
+          echo $?
+          /usr/bin/mc ls myminio/$MATTERMOST_BUCKET_NAME > /dev/null 2>&1
+          if [ $? -eq 1 ] ; then
+            echo "Creating bucket '$MATTERMOST_BUCKET_NAME'"
+            /usr/bin/mc mb myminio/$MATTERMOST_BUCKET_NAME
+          else
+            echo "Bucket '$MATTERMOST_BUCKET_NAME' already exists."
+            exit 0
+          fi
+
+Values that you need to replace in the above ``values.yaml`` file:
+
+- **<YOUR.MATTERMOST.DOMAIN>**: URL that users will use to access Mattermost, matching the `Site URL field <https://docs.mattermost.com/administration/config-settings.html#site-url>`_, e.g. ``mattermost.gitlab.example.com``.
```

Comment:
Is this our usual formatting for variable names in code blocks? 
 - Do we use dashes or periods to connect multiple words in a variable name?
 - Do we bold these names (**) or use in-line code (``)?

Reply:
@amyblais thoughts on this one?

Reply:
@cpanato  Is there a specific reason these are separated by periods ( . )?

If not I'll probably change them to underscore ( _ )

Reply:
this is the domain name, should be dots, this is how you access from your browser

Reply:
In that case I would leave them as dots and bolded.

Reply:
done

Reply:
@amyblais @cpanato Sorry for the confusion here, I was referring to the variable names, e.g. `**<GITLAB.APP.SECRET>**`.

According to our [text formatting guidelines](https://docs.mattermost.com/process/documentation-guidelines.html#text-formatting), parameter values are formatted lowercase in monospace. 

![image](https://user-images.githubusercontent.com/13119842/50061890-dbad2300-0171-11e9-8384-5c798ed65792.png)

So I've updated these accordingly. If you have any questions or concerns, let me know.

PS: I agree that `mattermost.gitlab.example.com` should have dots, though that's an example rather than a parameter value.